react/features/welcome/components/WelcomePage.native.js

```diff
@@ -346,7 +345,7 @@ class WelcomePage extends AbstractWelcomePage<*> {
         return (
             <>
                 <JitsiStatusBar />
-                <LocalVideoTrackUnderlay style = { styles.welcomePage }>
+                <View style = { styles.welcomePage }>
```

Comment:
Do we need this view? Maybe we can combine these 2 views?

Reply:
I will try.

Reply:
Done.